orchestrator/packages/api/src/controllers/utils.ts

```diff
@@ -14,6 +14,15 @@ export const errorResponse = (
   return res.status(status).json({ errors: errors });
 };
 
+export const formatValidationError = (instancePath: string, message: string | undefined): string => {
+  let result = '';
```

Comment:
This all boils down to 
```
`${instancePath} ${message}`.trim()
```

Reply:
Not sure on `message`; the docs don't provide a valid type for the `errors` so I have no reference for when is defined vs not.

For `instancePath`, it is provided when some sub-schema is failing (e.g., a missing property on a given object under `script`).

Reply:
https://ajv.js.org/api.html#error-objects:
```
Each reported error is an object with the following properties:

interface ErrorObject {
  keyword: string // validation keyword.
  instancePath: string // JSON Pointer to the location in the data instance (e.g., `"/prop/1/subProp"`).
  ...
  message?: string // the error message (can be excluded with option `messages: false`).
  ...
}
```
so it looks like it should always be present if you don't omit messages.  Regardless, it seems like `?? ''` is an acceptable workaround.